docs/css/README.md

```diff
@@ -1,15 +1,13 @@
 # CSS
 
-This boilerplate uses PostCSS as a CSS preprocessor with a few utility plugins
-to make it "batteries included".
+This boilerplate uses Styled Components allowing you to write your CSS in your JavaScript,
```

Comment:
nit: write `styled-components` (including the code block!) and link to the s-c repo